packages/cryptoplease/lib/features/incoming_split_key_payments/src/bl/iskp_bloc.dart

```diff
@@ -50,12 +53,16 @@ class ISKPBloc extends Bloc<_Event, _State> {
   final CryptopleaseClient _client;
   final Ed25519HDKeyPair _account;
   final TxSender _txSender;
+  final BalancesBloc _balancesBloc;
 
   EventHandler<_Event, _State> get _handler => (event, emit) => event.map(
         create: (e) => _onCreate(e, emit),
         process: (e) => _onProcess(e, emit),
       );
 
+  void _refreshBalances() =>
```

Comment:
Let's rather avoid coupling `ISKPBloc` and `BalancesBloc`, the less BLoCs know about each other the better.

This glue functionality should rather be solved by a `BlocListener` of the `ISKPBloc` that sends the corresponding event to the `BalancesBloc`.